packages/design-system-react-native/package.json

```diff
@@ -19,19 +19,23 @@
   "exports": {
     ".": {
       "import": {
-        "types": "./dist/index.d.ts",
-        "default": "./dist/index.js"
+        "types": "./dist/index.d.mts",
+        "default": "./dist/index.mjs"
+      },
+      "require": {
+        "types": "./dist/index.d.cts",
+        "default": "./dist/index.cjs"
       }
     },
     "./package.json": "./package.json"
   },
-  "main": "./dist/index.js",
-  "types": "./dist/index.d.ts",
+  "main": "./dist/index.cjs",
+  "types": "./dist/index.d.cts",
   "files": [
     "dist/"
   ],
   "scripts": {
-    "build": "node scripts/build.js",
+    "build": "yarn generate-icons && ts-bridge --project tsconfig.build.json --verbose --clean --no-references && cp -r src/components/Icon/assets dist/components/Icon/",
```

Comment:
Aligns the build script with other monorepo packages: generate icons, transpile TypeScript using ts-bridge, and copy icon assets to the distribution directory.